packages/document-drive/src/server/builder.ts

```diff
@@ -0,0 +1,141 @@
+import { DocumentModel } from "document-model/document";
+import { ICache } from "../cache";
+import InMemoryCache from "../cache/memory";
+import { BaseQueueManager } from "../queue/base";
+import { IQueueManager } from "../queue/types";
+import { MemoryStorage } from "../storage/memory";
+import { IDriveStorage } from "../storage/types";
+import { DocumentDriveServer } from "./base-server";
+import { DefaultEventEmitter } from "./event-emitter";
+import { ListenerManager } from "./listener/listener-manager";
+import TransmitterFactory from "./listener/transmitter/factory";
+import SynchronizationManager from "./sync-manager";
+import {
+  DefaultListenerManagerOptions,
+  type DocumentDriveServerOptions,
+  type IEventEmitter,
+  type IListenerManager,
+  type ISynchronizationManager,
+  type ITransmitterFactory,
+} from "./types";
+
+/**
+ * Builder class for constructing BaseDocumentDriveServer instances with proper configuration
+ */
+export class DocumentDriveServerBuilder {
```

Comment:
It would be good to already name this ReactorBuilder to starting migrating the naming from DocumentDriveServer

Reply:
done